app/code/core/Mage/Catalog/Model/Resource/Abstract.php

```diff
@@ -584,8 +584,12 @@ public function getAttributeRawValue($entityId, $attribute, $store)
     {
         if (!$entityId || empty($attribute)) {
             return false;
-        }
+	}
```

Comment:
Looks like you replaced spaces with a tab.

Reply:
my fault, removed